fix qmake rpath in meshlab.pro and meshlabserver.pro
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 meshlab (1.3.3) precise; urgency=low
 .
   * new snapshot release
Author: Stefan Schlager <zarquon42@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- meshlab-1.3.3.orig/meshlab/src/meshlab/meshlab.pro
+++ meshlab-1.3.3/meshlab/src/meshlab/meshlab.pro
@@ -136,11 +136,11 @@ win32-g++:LIBS        	+= -L../external/
 #}
 
 linux-g++:LIBS += -L../external/lib/linux-g++ -ljhead -L../distrib -lcommon -lGLU
-linux-g++:QMAKE_RPATHDIR += ../distrib
+linux-g++:QMAKE_RPATHDIR += /usr/lib/meshlab
 linux-g++-32:LIBS += -L../external/lib/linux-g++-32 -ljhead -L../distrib -lcommon -lGLU
-linux-g++-32:QMAKE_RPATHDIR += ../distrib
+linux-g++-32:QMAKE_RPATHDIR += /usr/lib/meshlab
 linux-g++-64:LIBS += -L../external/lib/linux-g++-64 -ljhead -L../distrib -lcommon -lGLU
-linux-g++-64:QMAKE_RPATHDIR += ../distrib
+linux-g++-64:QMAKE_RPATHDIR += /usr/lib/meshlab
 
 # uncomment in your local copy only in emergency cases.
 # We should never be too permissive
--- meshlab-1.3.3.orig/meshlab/src/meshlabserver/meshlabserver.pro
+++ meshlab-1.3.3/meshlab/src/meshlabserver/meshlabserver.pro
@@ -26,11 +26,11 @@ win32-msvc2010:  LIBS += -L../distrib -l
 win32-msvc2012:  LIBS += -L../distrib -lcommon
 win32-g++:  LIBS += -L../distrib -lcommon
 linux-g++:  LIBS += -L../distrib -lcommon
-linux-g++:QMAKE_RPATHDIR += ../distrib
+linux-g++:QMAKE_RPATHDIR += /usr/lib/meshlab
 linux-g++-32:  LIBS += -L../distrib -lcommon
-linux-g++-32:QMAKE_RPATHDIR += ../distrib
+linux-g++-32:QMAKE_RPATHDIR += /usr/lib/meshlab
 linux-g++-64:  LIBS += -L../distrib -lcommon
-linux-g++-64:QMAKE_RPATHDIR += ../distrib
+linux-g++-64:QMAKE_RPATHDIR += /usr/lib/meshlab
 #CONFIG (release,debug | release) {
 #	win32-msvc2005:release:  LIBS += -L../common/release -lcommon
 #	win32-msvc2008:release:  LIBS += -L../common/release -lcommon
